From f67997129c3d3b59fca8ee0d45c9539dbce1e4c1 Mon Sep 17 00:00:00 2001
From: Bai Ping <b51503@freescale.com>
Date: Wed, 22 Jul 2015 00:14:34 +0800
Subject: [PATCH] MLK-11270 ARM: imx: correct the pmic standby voltage of
 sw1ab on i.mx6ul

The SW1AB on PMIC is used for ARM_SOC_IN supply, set the standby voltage to
0.975V to save power when system is in DSM mode.

Signed-off-by: Bai Ping <b51503@freescale.com>
---
 board/freescale/mx6ul_14x14_ddr3_arm2/mx6ul_14x14_ddr3_arm2.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/board/freescale/mx6ul_14x14_ddr3_arm2/mx6ul_14x14_ddr3_arm2.c b/board/freescale/mx6ul_14x14_ddr3_arm2/mx6ul_14x14_ddr3_arm2.c
index bb78c7b..46e7cd1 100644
--- a/board/freescale/mx6ul_14x14_ddr3_arm2/mx6ul_14x14_ddr3_arm2.c
+++ b/board/freescale/mx6ul_14x14_ddr3_arm2/mx6ul_14x14_ddr3_arm2.c
@@ -825,10 +825,10 @@ int power_init_board(void)
 	 * them, when needed, configure them to our requested voltage.
 	 */
 
-	/* set SW1AB standby volatage 1.3V */
+	/* set SW1AB standby volatage 0.975V */
 	pmic_reg_read(pfuze, PFUZE100_SW1ABSTBY, &value);
 	value &= ~0x3f;
-	value |= PFUZE100_SW1ABC_SETP(13000);
+	value |= PFUZE100_SW1ABC_SETP(9750);
 	pmic_reg_write(pfuze, PFUZE100_SW1ABSTBY, value);
 
 	/* set SW1AB/VDDARM step ramp up time from 16us to 4us/25mV */
-- 
1.8.0

